pkg/handler/pullrequest/handler.go

```diff
@@ -34,18 +34,23 @@ func Handle(pr github.PullRequestPayload) error {
 	gitRepo.Lock()
 	defer gitRepo.Unlock()
 
+	config, err := repoconfig.Read(gitRepo, pr.PullRequest.Base.Sha)
+	if err != nil {
+		klog.Infof("Error reading bot config: %s", err)
+	}
+
```

Comment:
For simplicity's sake you can keep it where it was (inside `openOrSync`)

Reply:
oh right